builder/package.ubuntu-1804

```diff
@@ -0,0 +1,50 @@
+if [[ ! -d /tmp/output/ubuntu-1804 ]]; then
+  mkdir /tmp/output/ubuntu-1804
```

Comment:
I got a "mkdir: cannot create directory" error when I was testing this out at first, and had to add a `-p` flag here

Reply:
✅ Done